### PR TITLE
feat: ✨ New Extensions: Oh My Zsh Git Alias

### PR DESCRIPTION
**Pull request by yufeikang**
## Description

Quick search oh-my-zsh git plugin alias

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by peduarte** [comment]
Sorry, I'll re-open this and take a look!

File: extensions/ohmyzsh-git-alias/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "name": "index",
+      "title": "Git Alias",
+      "description": "Search Oh-My-Zsh Git Alias",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.45.2",
+    "axios": "^1.2.1"
```
Comment (peduarte) [inline extensions/ohmyzsh-git-alias/package.json:23]: This doesnt look like it's being used, can we remove and ensure pacakge-lock is updated. Thanks!

File: extensions/ohmyzsh-git-alias/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const [searchText, setSearchText] = useState<string>("");
+
+  const onSearchTextChange = (text: string) => {
+    setSearchText(text);
+  };
+
+  return (
+    <List onSearchTextChange={onSearchTextChange}>
+      {alias.map((a: { alias: string; command: string }) => {
+        if (searchText !== "" && !a.alias.includes(searchText) && !a.command.includes(searchText)) {
```
Comment (peduarte) [inline extensions/ohmyzsh-git-alias/src/index.tsx:15]: Is there a reason you are checking for this? Raycast should handle search manually pretty well!

File: extensions/ohmyzsh-git-alias/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            key={a.alias}
+            title={a.alias}
+            subtitle={a.command}
+            actions={
+              <ActionPanel title="Copy">
+                <Action.CopyToClipboard title="Copy Alias" content={a.alias} />
+                <Action.CopyToClipboard title="Copy Command" content={a.command} />
+              </ActionPanel>
+            }
+          />
```
Comment (peduarte) [inline extensions/ohmyzsh-git-alias/src/index.tsx:29]: Might also be worth adding a `Action.Paste` (https://developers.raycast.com/api-reference/user-interface/actions#action.paste) Action, so users can paste it directly :)

File: extensions/ohmyzsh-git-alias/src/index.tsx (hunk 2 above)
Comment (yufeikang) [reply extensions/ohmyzsh-git-alias/src/index.tsx:15]: It looks like raycast can't search for subtitle.

File: extensions/ohmyzsh-git-alias/package.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
   "description": "Quick search git alias.",
   "icon": "icon.png",
   "author": "yufeikang",
+  "owner": "kang",
```
Comment (peduarte) [inline extensions/ohmyzsh-git-alias/package.json]: You shouldn't need the `owner` field unless youre an organization

File: extensions/ohmyzsh-git-alias/src/index.tsx (hunk 2 above)
Comment (peduarte) [reply extensions/ohmyzsh-git-alias/src/index.tsx:15]: you can use the `keywords` prop, the search takes that into account and is hidden from the UI. That way you can still rely on fuzzy search :)

Comment (peduarte) [reply extensions/ohmyzsh-git-alias/src/index.tsx:15]: https://developers.raycast.com/api-reference/user-interface/list#list.item

File: extensions/ohmyzsh-git-alias/package.json (hunk 4 above)
Comment (yufeikang) [reply extensions/ohmyzsh-git-alias/package.json]: I found this. I have removed this. [commit](https://github.com/raycast/extensions/pull/4062/commits/6d3bac8ad9a4104e6040da8d0e5bca0ccfb43cd3)

File: extensions/ohmyzsh-git-alias/src/index.tsx (hunk 2 above)
Comment (yufeikang) [reply extensions/ohmyzsh-git-alias/src/index.tsx]: Awesome. Thank you.

**Comment by peduarte** [comment]
Great stuff! 🚢

**Pull request merged**